### ISSUE TITLE
RandomX options

### ISSUE THREAD

**Issue by Lokir123** [UnamSanctam/SilentCryptoMiner#2353]
How do I set up RandomX mining mode: "auto", "fast" (2 GB memory), "light" (256 MB memory)?
I need to set the "light" mode

**Comment by Lokir123** [comment]
Why can't I edit the Json window?

**Comment by UnamSanctam** [comment]
You can do it using the "Advanced Parameters" field, you can add `--randomx-mode=light` into it which should do what you want https://xmrig.com/docs/miner/command-line-options.

>Why can't I edit the Json window?

Because it only shows you your current miner settings in a JSON format, mainly for use with the "Remote Configuration" feature or for a configuration on the web panel.

**Comment by Lokir123** [comment]
Thanks for the quick answers, kind man. Another question is, why are you doing this for free?) Or do you get paid for it somehow

**Comment by UnamSanctam** [comment]
For fun mostly, I work on it during my free time to get away from work. Though I do get donations from kind people sometimes.

**Comment by Lokir123** [comment]
You could add donations to your wallet. Why don't you do that?

**Comment by UnamSanctam** [comment]
I don't really need it, and it would make the project more like a job instead of a hobby.

**Comment by Lokir123** [comment]
Can I embed 2 xmr miners? What would they mine on 2 different wallets

**Comment by UnamSanctam** [comment]
Yes you can do that, and yes if you enter one wallet address into one and another into the other one then they'll mine separately to those two wallet addresses.

**Comment by UnamSanctam** [comment]
>Your free time feels like my entire day. I swear you have more hours in a day than the rest of us, or you can pause time or something. Whenever I get a notification from these repositories, even just 2 minutes ago, I can be sure that you have already answered. And even if there is an issue that requires you to code, done in 4 minutes, on your phone even... maybe you are an immortal, you were there when Unam Sanctam was issued in 1302. Haha jk, but why did you pick UnamSanctam as your username?

Haha thank you, I guess I'm just usually at some device where I can answer most of the time. I chose Unam Sanctam since it's a very interesting part of history (for me at least).
